docs/introduction/path_to_chrysalis.md

```diff
@@ -8,81 +8,77 @@ Progress in Coordicide research has resulted in identifying many concepts that c
 
-Another important requirement is the ease of transition to the future Coordicide network. By carefully planning the breaking changes introduced along the way, and providing support in our developer tools, we will ensure that our growing ecosystem of developers, startups, and corporations can reliably develop and launch new innovative products on IOTA.  
+Another important requirement was the ease of transition to the future Coordicide network. By carefully planning the breaking changes introduced along the way, and providing support in our developer tools, we ensured that our growing ecosystem of developers, startups, and corporations could reliably develop and launch new innovative products on IOTA.  
 
-The plan for implementing Chrysalis is divided into two phases.
+The plan for implementing Chrysalis was divided into two phases.
 
 **The first phase** consisted of improved tip selection (URTS), milestone selection, and White flag. These were implemented in the node software gradually. This phase required an upgrade of all nodes, including the coordinator node, and did not require a snapshot.  
 
 The first phase of Chrysalis resulted in:
 
-- Transaction confirmation times of around 10 seconds
-- Transactions rarely needing reattachment
-- A substantial TPS increase
-- Performance and reliability improvements for nodes
+- Transaction confirmation times of around 10 seconds.
+- Transactions rarely needing reattachment.
+- A substantial TPS increase.
+- Performance and reliability improvements for nodes.
 
-**The second phase** of Chrysalis consists of adopting and/or implementing UTXO, atomic transactions, reusable addresses (Ed25519), a transition to a binary transaction layout, and a new set of client libraries and developer tools. These represent significant changes to the core protocol and the way transactions are structured. Once everything has been tested, validated, and audited, the Foundation will deploy a new Chrysalis network. The upgrade will consist of an extended period when the current, legacy network remains operational. This allows users, exchanges, and partners to migrate to the Chrysalis network at will. The migration is not time constrained.  
+**The second phase** of Chrysalis consisted of adopting and/or implementing UTXO, atomic transactions, reusable addresses (Ed25519), a transition to a binary transaction layout, and a new set of client libraries and developer tools. These represented significant changes to the core protocol and the way transactions were structured. Once everything was tested, validated, and audited, the Foundation deployed a new Chrysalis network. The upgrade consisted of an extended period when the legacy network remained operational which allowed users, exchanges, and partners to migrate to the Chrysalis network at will. 
```

Comment:
- Did we adopt, implement or both ? 
- We could clarify that UTXO stands for Unspent transaction output, and add a link to https://github.com/iotaledger/stronghold.rs/tree/dev/.changes
- We could add a link to the Ed25519, reusable addreses and such when we mention them. 
- "These represented", is these the new set of client libraries and dev tools? Or all the upgrades in general?